src/pre_defined_classes/simple_programs_taking_numeric_user_input/hard/q10/NewSalaryCalculator.java

```diff
@@ -0,0 +1,33 @@
+package pre_defined_classes.simple_programs_taking_numeric_user_input.hard.q10;
+/*
+A company called HappyWorkLife has decided to give everybody a raise with an increase of 5% for each year of
+their service. "The raise only happen to those who have served at least a year or more,
+and the raise is calculated annually." Make a program that asks the user for the salary and the number of years worked
+at the company, then display the increased salary amount.
```

Comment:
Yeah, I think it is good to add this, I would just also add that the salary and years should be positive. It could say something like `Assume the user enters non-negative integers and that the raise is calculated annually.`